content/posts/03_system_desing_interview/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "How do you conduct and evaluate a system design interview?"
+date: 2023-02-26
+tags: [
+    "interview",
+]
+draft: true,
+---
+
+1. Intro
+  * different approaches, based on the company requirements
+  * helps to evaluate and distinguish senior and staff+ candidates
+  * difficult to do well
+2. Goals of the sys design interview
+  * High level goals the same as for all interviews
+  * Would the candidate be able to do their job
+  * Which of our career levels does the candidate fit best into
+  * Would the candidate raise our bar of talents
+3. Competencies and skills to evalute
+  * prbolem understanding
+  * getting it done
+  * distributed systems knowledge
+  * technical leadership
+4. Pick and prepare a system 
+  * something you know and understand well
+  * split into increments and outline potential interview framework (not that candaidates may not follow this exactly and it's fine)
+  * think about limitations, use cases, current and future load
```

Comment:
I'd say:

"have a clear functional and technical requirements in mind"

Reply:
n/a, just outline